fe/fe-core/src/main/java/com/starrocks/planner/RuntimeFilterDescription.java

```diff
@@ -180,26 +184,21 @@ public boolean canPushAcrossExchangeNode() {
             return false;
         }
         switch (joinMode) {
-            case BROADCAST: {
-                // Broadcast HJ generates non-shuffle-aware(single-part) rf, so it can be always
-                // pushed down across ExchangeNode.
-                return true;
-            }
+            case BROADCAST:
             case PARTITIONED:
             case LOCAL_HASH_BUCKET:
             case SHUFFLE_HASH_BUCKET:
-            case COLOCATE: {
-                // these HJ always generates shuffle-aware(multi-part) rf, at present only rf generated by
-                // HJ with only one equalConjunct is allowed to pushed down across ExchangeNode.
-                // TODO(by satanson): rf generated by HJ with multiple equalConjuncts needs partition_by_exprs
-                //  to choose the right partial component of multi-part global rf.
-                return equalCount == 1;
-            }
+            case COLOCATE:
```

Comment:
are all types true?

Reply:
BUCKET_SHFUFLE is false

Reply:
BUCKET_SHFUFLE looks like true now

Reply:
Replica is false， and I think there should be strict check